xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -255,28 +302,92 @@ extern "C" {
+    int          iLifetimesDefault;                         /*!< @brief (optional) The default value for PVR_TMER.iLifetime. Must be filled if iLifetimesSize > 0 */
+
+    /* prevent duplicate episodes value definitions */
+    unsigned int iPreventDuplicateEpisodesSize;             /*!< @brief (required) Count of possible values for PVR_TMER.iPreventDuplicateEpisodes. 0 means duplicate
+                                                              episodes prevention is not supported by this timer type or no own value definition wanted, but to use
+                                                              Kodi defaults. */
+    PVR_TIMER_TYPE_ATTRIBUTE_INT_VALUE
+      preventDuplicateEpisodes[PVR_ADDON_TIMERTYPE_VALUES_ARRAY_SIZE];
+                                                            /*!< @brief (optional) Array containing the possible values for PVR_TMER.iPreventDuplicateEpisodes.. Must
+                                                              be filled if iPreventDuplicateEpisodesSize > 0 */
+    unsigned int iPreventDuplicateEpisodesDefault;          /*!< @brief (optional) The default value for PVR_TMER.iPreventDuplicateEpisodesSize. Must be filled if iPreventDuplicateEpisodesSize > 0 */
+
+    /* recording folder list value definitions */
+    unsigned int iRecordingGroupSize;                       /*< @brief (required) Count of possible values of PVR_TIMER.iRecordingGroup. 0 means folder lists are not supported by this timer type */
+    PVR_TIMER_TYPE_ATTRIBUTE_INT_VALUE
+      recordingGroup[PVR_ADDON_TIMERTYPE_VALUES_ARRAY_SIZE];
+                                                            /*!< @brief (optional) Array containing the possible values of PVR_TMER.iRecordingGroup. Must be filled if iRecordingGroupSize > 0 */
+    unsigned int iRecordingGroupDefault;                    /*! @brief (optional) The default value for PVR_TIMER.iRecordingGroup. Must be filled in if PVR_TIMER.iRecordingGroupSize > 0 */
+
+  } ATTRIBUTE_PACKED PVR_TIMER_TYPE;
+
+  /*!
    * @brief Representation of a timer event.
    */
   typedef struct PVR_TIMER {
     unsigned int    iClientIndex;                              /*!< @brief (required) the index of this timer given by the client */
-    int             iClientChannelUid;                         /*!< @brief (required) unique identifier of the channel to record on */
-    time_t          startTime;                                 /*!< @brief (required) start time of the recording in UTC. instant timers that are sent to the add-on by xbmc will have this value set to 0 */
-    time_t          endTime;                                   /*!< @brief (required) end time of the recording in UTC */
+    unsigned int    iParentClientIndex;                        /*!< @brief (optional) for timers scheduled by a repeating timer, the index of the repeating that scheduled this timer. 0 indicates "no parent". */
```

Comment:
`the index of the repeating that scheduled this timer`
think something's missing here ;-)

can you please check all the descriptions that you've added or changed in the api files, so an add-on author knows what to do with each field exactly. the other files changed are less important, but this one is ;-)